packages/story-editor/src/app/highlights/quickActions/useQuickActions.js

```diff
@@ -465,7 +465,12 @@ const useQuickActions = () => {
   );
 
   const videoCommonActions = useMemo(() => {
-    return hasTrimMode
+    const resource = selectedElements?.[0]?.resource;
+    if (!resource) {
+      return [];
+    }
+    const { isTrimming, isMuting, isTranscoding } = resource;
+    return !isTrimming && !isMuting && !isTranscoding && hasTrimMode
```

Comment:
This is going to change in https://github.com/google/web-stories-wp/pull/9098

Is this check for isTrimming etc part of hasTrimMode?
https://github.com/google/web-stories-wp/blob/36f34ea5d9995fb4c5f38d68cb605ab6e6c87a2c/packages/story-editor/src/components/videoTrim/useVideoTrimMode.js#L59-L74

Reply:
> Is this check for isTrimming etc part of hasTrimMode?

It was but I removed it since otherwise the button gets hidden when `isTrimming` but it should just be disabled.

Reply:
If that is the case, can you code review #9098

Reply:
@spacedmonkey This is a good place to use `canTranscodeResource` that you added in #9098.